dist/livewire.js

```diff
@@ -7624,6 +7624,7 @@ ${expression ? 'Expression: "' + expression + '"\n\n' : ""}`, el);
   var oldBodyScriptTagHashes = [];
   var attributesExemptFromScriptTagHashing = [
     "data-csrf",
+    "nonce",
```

Comment:
`dist` folder is contains all scripts that already compiled. Maybe you mean this file?
https://github.com/livewire/livewire/blob/main/js/plugins/navigate/page.js

Try run `npm run watch` to see if the problem is fixed, if it fixed then run `npm run build` to compile all the assets.

Can you also make a test if the problem is fixed? if it's related with `wire:navigate` you can make the test [here](https://github.com/livewire/livewire/blob/main/src/Features/SupportNavigate/BrowserTest.php)

Reply:
my bad, thank you for pointing me in the right direction